File: Q2180.c
int countEven(int num) {
    int count=0,sum=0;

    for(int i=1;i<=num;i++) {
        sum=0;
      for(int j=i;j>0;j=j/10) {
        sum=sum+(j%10);
      }  
      if(sum%2==0)
      count++;
    }
    return count;
}

File: Q2180.py
class Solution:
    def countEven(self, num: int) -> int:
        count=0

        for i in range(1,num+1):
            sum=0
            for j in str(i):
                sum=sum+int(j)
                i=i/10
            if sum%2==0:
                count=count+1      
      
        return count
